scripts/lib.sh

```diff
@@ -1969,7 +1969,7 @@ function set_core_random_samples () {
     fi
 
     if [ -z "$SAMPLE_GENE" ]; then
-      local RAND_GENE=$($CMD -D "$DBNAME" -N -e 'select stable_id from gene order by rand() limit 1;')
+      local RAND_GENE=$($CMD -D "$DBNAME" -N -e 'SELECT stable_id FROM gene g LEFT JOIN seq_region sr ON g.seq_region_id = sr.seq_region_id LEFT JOIN coord_system USING (coord_system_id) where g.biotype = "protein_coding" AND coord_system.name = "primary_assembly" ORDER BY RAND() LIMIT 1;')
```

Comment:
Filtering on biotype makes sense, 
"primary_assembly" -- is not working for older assemblies.
We need to check for the top_level seq_region_attrib, or just safely ignore, because we LoadGFF3 places genes only on top-level seq regions.

Reply:
I've updated the query to link to seq_region_attrib restricting to those genes linked to `Top Level`. Would that be ok ?